### ISSUE TITLE
NTR: Tick species

### ISSUE THREAD

**Issue by SAK22** [VEuPathDB-ontology/VEuPathDB-ontology#487]
Please could we add the following tick species terms.

Candidatus VBsp:0600094
Neoehrlichia VBsp:0600095
Neoehrlichia mikurensis VBsp:0600096
Borrelia afzelii VBsp:0600104
Borrelia garinii VBsp:0600103
Borreliella burgdoferi sensu lato VBsp:0600102
Borreliella lusitaniae VBsp:0600093
Borreliella spielmanii VBsp:0600092
Borreliella valaisiana VBsp:0600091
Spotted fever group VBsp:0600100
Rickettsia helvetica VBsp:0600101

**Comment by johnwjudkins** [comment]
@SAK22 It looks like Candidatus is not a taxon but an indicator of a type of prokaryote that has not yet been cultured and has a provisional name. Do we want a term for any prokaryote that has this designation? Or does Candidatus mean something else in this list?

**Comment by johnwjudkins** [comment]
Also, is it "burgdorferi" rather than "burgdoferi"?

**Comment by SAK22** [comment]
Yes to burgdorferi! And I am not sure about Candidatus, let me look at the paper it came from and link it here: https://www.nature.com/articles/s41597-020-00579-y it is reported in the supplemtary files that I am using to populate the map.
